sos/report/plugins/instructlab.py

```diff
@@ -0,0 +1,130 @@
+        # We gather this always.
+        local_share_dir = '/.local/share/instructlab'
+
+        # container paths
+        cont_cache_path = f'{cont_opt_path}{cache_dir}'
+        cont_config_path = f'{cont_opt_path}{config_dir}'
+        cont_local_path = f'{cont_opt_path}{local_share_dir}'
+
+        self.add_forbidden_path([
+            f"{cont_local_path}/taxonomy/.git",
+            f"{cont_local_path}/taxonomy/.github",
+            f"{cont_opt_path}/src/.local/share/instructlab/taxonomy/.git",
+            f"{cont_opt_path}/src/.local/share/instructlab/taxonomy/.github",
+            ])
+
+        subcmds = [
+            'taxonomy diff',
+            'taxonomy diff --taxonomy-base=empty',
+            'system info',
+            'model list',
+            'config show'
+        ]
+
+        data_dirs = [
+            'data',
+            'generated',
+            'taxonomy',
+            'taxonomy_data',
+            'chatlogs',
+            'checkpoints',
+            'datasets',
+            'internal',
+            'phased',
+        ]
+
+        in_container = False
+        container_names = []
+        _containers = self.get_containers()
+
+        for _con in _containers:
+            if _con[1].startswith('instructlab'):
+                in_container = True
+                container_names.append(_con[1])
+
+        if in_container:
+            for cont in container_names:
+                self.add_copy_spec(
+                    [f'{cont_share_conf_path}rhel_ai_config.yaml',
+                     f'{cont_config_path}/config.yaml'],
+                    container=cont)
+                self.add_copy_spec(
+                    [f"{cont_local_path}/{data_dir}"
+                     for data_dir in data_dirs],
+                    container=cont
+                    )
+                self.add_cmd_output(
+                    [f"ilab {sub}" for sub in subcmds],
+                    container=cont
+                )
+                self.add_cmd_output(f"ls -laR {cont_cache_path}",
+                                    container=cont)
+                if self.get_option("get-cache"):
+                    self.add_copy_spec(
+                        f'{cont_cache_path}',
+                        container=cont
+                        )
+                self.add_container_logs(cont)
+
+        ilab_user = self.get_option("ilab_user") if \
+            self.get_option("ilab_user") else ''
+        if ilab_user:
+            ilab_dir = f'/home/{ilab_user}'
+            if self.get_option("ilab_conf_dir"):
+                ilab_dir = f'{ilab_dir}{self.get_option("ilab_conf_dir")}'
+            data_dirs_base = f'{ilab_dir}{local_share_dir}'
+
+            self.add_copy_spec(f"{ilab_dir}{config_dir}")
+            self.add_copy_spec([
+                f"{data_dirs_base}/{data_dir}" for data_dir in data_dirs
+            ])
+            self.add_cmd_output(f"ls -laR {ilab_dir}/{cache_dir}")
```

Comment:
same here, can use `self.add_dir_listing()`

Reply:
And changing this one as well